.github/workflows/pr-triage.yml

```diff
@@ -0,0 +1,73 @@
+name: PR Triage
+
+on: pull_request
+
+jobs:
+  add_pkg_labels:
+    name: "Add \"pkg: *\" labels"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get corresponding labels
+        id: labels
+        uses: babel/actions/get-modified-files-labels@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          pull_number: ${{ github.event.pull_request.number }}
+          labels: >
+            {
```

Comment:
Could it be extracted as a dedicated JSON/yaml configuration? It could be similar to what it is done in [Labeller](https://github.com/actions/labeler#create-githublabeleryml). 

By doing so It will benefit from better error check / prettier styling and I would expect it can be more visible to contributors.

Reply:
Oh well, I could probably use Labeller directly